ui/src/components/release.vue

```diff
@@ -14,55 +14,59 @@
-        <b-badge pill class="bigpill clickable" @click="downloadDataset(release.set)" style="float: right; background-color: #2693ff;  color: white;">
-            <icon name="download" style="opacity: 0.4;"/>&nbsp;&nbsp;&nbsp;<small>Dowload</small>
-        </b-badge>
-        <p style="margin-bottom: 5px;"><small>The following data objects are published as part of this release.</small></p>
-        <div style="border-top: 1px solid #eee; padding: 3px 0;" v-for="(set, idx) in release.sets" :key="idx">
-            <b-row>
-                <b-col cols="5">
-                    <datatypetag :datatype="set.datatype" :tags="set.datatype_tags" :clickable="false"/>
-                    <small v-if="set.subjects && set.subjects.length == 0">({{set.count||0}} obj <span style="opacity:0.5">|</span> {{(set.size||0)|filesize}})</small>
-                </b-col>
-                <b-col>
-                    <p style="margin-bottom: 0px; line-height: 150%"><small>{{set.datatype.desc}}</small></p>
-                </b-col>
-            </b-row>
+    <a :name="'release.'+release.name+'.data'">
+        <div v-if="release.sets && release.sets.length">
+            <span class="subheader">Data</span>
+            <b-badge pill class="bigpill clickable" @click="downloadDataset(release.set)" style="float: right; background-color: #2693ff;  color: white;">
+                <icon name="download" style="opacity: 0.4;"/>&nbsp;&nbsp;&nbsp;<small>Dowload</small>
+            </b-badge>
+            <p style="margin-bottom: 5px;"><small>The following data objects are published as part of this release.</small></p>
+            <div style="border-top: 1px solid #eee; padding: 3px 0;" v-for="(set, idx) in release.sets" :key="idx">
+                <b-row>
+                    <b-col cols="5">
+                        <datatypetag :datatype="set.datatype" :tags="set.datatype_tags" :clickable="false"/>
+                        <small v-if="set.subjects && set.subjects.length == 0">({{set.count||0}} obj <span style="opacity:0.5">|</span> {{(set.size||0)|filesize}})</small>
+                    </b-col>
+                    <b-col>
+                        <p style="margin-bottom: 0px; line-height: 150%"><small>{{set.datatype.desc}}</small></p>
+                    </b-col>
+                </b-row>
+            </div>
+            <br>
         </div>
-        <br>
-    </div>
-
-    <div v-if="release.apps && release.apps.length">
-        <span class="subheader">Preprocessing</span>
-        <small>The following Apps were used to generate the data in this release.</small>
-        <div v-for="rec in release.apps" :key="rec._id" style="margin-top: 10px; border-left: 3px solid #f0f0f0; border-bottom: 1px solid #eee;">
-            <app :appid="rec.app" :branch="rec.task.service_branch" :compact="true" :showDoi="true">
-                <taskconfig :task="rec.task" style="margin: 10px;"/>
-            </app>
+    </a>
+    <a :name="'release.'+release.name+'.preprocessing'">
```

Comment:
Same here. I am afraid that style for `<a>` would somehow contaminate the CSS rules for other things.